<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cutterTab</class>
 <widget class="QWidget" name="cutterTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>696</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
   <item>
    <widget class="QGroupBox" name="canvasGroup">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>false</bold>
       <underline>false</underline>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="title">
      <string>холст</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_5"/>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="tableGroup">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>false</bold>
       <underline>false</underline>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="title">
      <string>Инспектор</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>1000</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <bold>false</bold>
          <underline>false</underline>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="text">
         <string>ЛКМ - добавление отрезков,
добавление отсекателя</string>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cutButton">
        <property name="text">
         <string>отсечь</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearButton">
        <property name="text">
         <string>сбросить отсечение</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="selectCutter">
        <property name="text">
         <string>выбрать отсекатель</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addRegionButton">
        <property name="text">
         <string>добавить многоугольник</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removeRegionButton">
        <property name="text">
         <string>удалить многоугольники</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
